README.md

```diff
@@ -1,4 +1,13 @@
 # @apostrophecms/event
 Note: This module is in Alpha and not ready for use. It is a community-supported bundle and contributions are welcome!
 
-This bundle is intended to provide event functionality (concerts, recurring webinars, etc.) with <a href="https://apostrophecms.com/">Apostrophe CMS 3</a>. It is based on the `apostrophe-events` bundle for Apostrophe CMS 2. 
+This bundle is intended to provide event functionality (concerts, recurring webinars, etc.) with <a href="https://apostrophecms.com/">Apostrophe CMS 3</a>. It is based on the `apostrophe-events` bundle for Apostrophe CMS 2.
+
+## Stable release punch list
+- [ ] The index page should only show upcoming published events
+- [x] Index page default template should use H2 and H3 headings for sections (filters and events) and a paragraph or `date` for the date.
+- [ ] Index page should have pagination.
+- [ ] needs localization keys
+- [ ] Make event description optional
+- [ ] Change `groupId` to `eventGroupId`
```

Comment:
`groupId` is a little too generic. The property should be more specific to this module.